src/main/rx/rx.c

```diff
@@ -559,10 +561,10 @@ FAST_CODE_NOINLINE void rxFrameCheck(timeUs_t currentTimeUs, timeDelta_t current
     } else {
         //  watch for next packet
         if (cmpTimeUs(currentTimeUs, needRxSignalBefore) > 0) {
-            //  initial time to signalReceived failure is 100ms, then we check every 100ms
+            //  initial time to rxDataReceived failure is RXLOSS_TRIGGER_INTERVAL, then we check every RX_FRAME_RECHECK_INTERVAL
             rxSignalReceived = false;
-            needRxSignalBefore = currentTimeUs + needRxSignalMaxDelayUs;
-            //  review and process rcData values every 100ms in case failsafe changed them
+            needRxSignalBefore = currentTimeUs + reCheckRxSignalInterval;
```

Comment:
Should this be `needRxSignalBefore += reCheckRxSignalInterval;`? That prevents any accumulation of timing error.

Reply:
Yes, that's better, thanks.